hrm-service/permission-rules/br.json

```diff
@@ -18,5 +18,6 @@
   "editChildIsAtRisk": [["isSupervisor"], ["isCreator", "isCaseOpen"]],
   "editFollowUpDate": [["isSupervisor"], ["isCreator", "isCaseOpen"]],
   "editContact": [["isSupervisor"]],
-  "viewPostSurvey": [["isSupervisor"]]
+  "viewPostSurvey": [["isSupervisor"]],
+  "viewExternalTranscript": [["isSupervisor"]]
```

Comment:
I'm curious on the naming of `viewExternalTranscript` vs just `viewTranscript`. I assume that "external" means something that we store (or, that is not stored in Twilio). Will there ever be a time when we want to differentiate between someone having permissions to view a Twilio-generated transcript versus viewing a transcript that we store outside of Twilio?

Reply:
After talking with @stephenhand about this, we agreed that it may useful to leave the door open for this behavior. We are already storing Twilio stored transcripts (which are accessed via task reservation sid). In [this PR](https://github.com/techmatters/flex-plugins/pull/979) you can see the differences in the frontend about who and when can access external or Twilio stored transcripts. Since the permissions of the Twilio stored one is derived from the permissions to access insights (as that's how Twilio handles transcripts), I didn't thought introducing that one here would be needed, but adding similar logic to handle that using this permissions files instead of relying on the `wfo` prefix of the worker roles should be very simple (the bases for that are done in this PR).

I'm happy to rename/change this if you still think is better!